surveys/2021-annual-survey/questions.md

```diff
@@ -123,9 +123,37 @@ Select all that apply:
 
 ## Your Rust experience
 
+### When did you learn to program in Rust?
+
+**Note**: while you may continue to try to improve your Rust skills, for this question assume "learning to program in Rust"
+means spending the *majority* of your time with Rust consuming learning materials or coding *in order to learn* (as opposed to achieving
+some other goal). If your learning process spans several of the listed time periods, pick the one where you felt you learned *the most*.
+
+Select one:
+
+- I'm still *actively* trying to learn Rust
+- During 2021
+- During 2019 or 2020
+- During 2015, 2016, 2017, or 2018
```

Comment:
The error messages [changed](https://blog.rust-lang.org/2016/09/29/Rust-1.12.html) quite a bit in late 2016 and thus learning Rust started looking much more similar to what it looks like today. We may want to distinguish between 2015/2016 and 2017/2018.